base-notebook/start.sh

```diff
@@ -16,6 +16,9 @@ if [ $UID == 0 ] ; then
                 chown -R $NB_UID "$d"
             fi
         done
+    elif [[ -d /home/$NB_USER/work && $(stat -c %U /home/$NB_USER/work) != "$NB_USER" ]]; then
```

Comment:
I meant, should we check if `/home/$NB_USER` has the right ownership here if the code inside the block is going to change that folder, and not `/home/$NB_USER/work`?